src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@use '../utils/variables' as *;
+@use '../utils/mixins' as *;
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
```

Comment:
The `$card-width` variable is used here. Ensure that it is defined in the `variables` file to avoid any compilation errors.